src/main/java/au/org/emii/netcdf/iterator/reader/NetcdfReader.java

```diff
@@ -76,4 +80,13 @@ public String getString(Set<IndexValue> indexValues) {
         }
     }
 
+    public String getDisplayValue(Set<IndexValue> indexValues) {
+        EnhanceScaleMissing scaleMissingDecorator = (EnhanceScaleMissing) variable;
+
+        if (scaleMissingDecorator.hasMissing() && scaleMissingDecorator.isMissing(getDouble(indexValues))) {
```

Comment:
Can set the enhanced variable to return NaN for missing values - but, not sure why I would do that when I can check missing properties on the enhanced variable here - its clearer I think.

I'll pull the condition out into a method describe what I'm doing better here.